.github/workflows/release.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release Rulesfile
+
+on:
+  push:
+    tags:
+      # potential semver
+      - '[a-z]*[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  release-rulesfile:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Rules
+        uses: actions/checkout@v3
+
+      - name: Setup Golang
+        uses: actions/setup-go@v3
+        with:
+          go-version: '^1.19'
+
+      - name: Build registry artifact tool
+        working-directory: build/registry
+        run: go build -o rules-registry ./...
+
+      - name: Get lowercase OCI repo prefix
+        run: |
+          echo "OCI_REPO_PREFIX=ghcr.io/${GITHUB_REPOSITORY,,}" >>${GITHUB_ENV}
```

Comment:
I didn't find a better way to get the lowercase repository name :sweat_smile: , this is not an issue for `falcosecurity` but it was while I was testing it in my account which is called `LucaGuerra`. I'd like to leave it here so that it's a bit more generic.